Mathlib/SetTheory/Cardinal/Ordinal.lean

```diff
@@ -209,41 +225,46 @@ theorem aleph'_omega : aleph' ω = ℵ₀ :=
     simp only [aleph'_le_of_limit omega_isLimit, lt_omega, exists_imp, aleph0_le]
     exact forall_swap.trans (forall_congr' fun n => by simp only [forall_eq, aleph'_nat])
 
+set_option linter.deprecated false in
 /-- `aleph'` and `aleph_idx` form an equivalence between `Ordinal` and `Cardinal` -/
-@[simp]
+@[deprecated aleph' (since := "2024-08-28")]
 def aleph'Equiv : Ordinal ≃ Cardinal :=
   ⟨aleph', alephIdx, alephIdx_aleph', aleph'_alephIdx⟩
 
-/-- The `aleph` function gives the infinite cardinals listed by their
-  ordinal index. `aleph 0 = ℵ₀`, `aleph 1 = succ ℵ₀` is the first
-  uncountable cardinal, and so on. -/
-def aleph (o : Ordinal) : Cardinal :=
-  aleph' (ω + o)
+/-- The `aleph` function gives the infinite cardinals listed by their ordinal index. `aleph 0 = ℵ₀`,
+`aleph 1 = succ ℵ₀` is the first uncountable cardinal, and so on.
+
+For a version including finite cardinals, see `Cardinal.aleph'`. -/
+def aleph : Ordinal ↪o Cardinal :=
+  OrderEmbedding.ofMapLEIff
+    (fun o => aleph' (ω + o)) fun _ _=> aleph'_le.trans (add_le_add_iff_left ω)
```

Comment:
Can you build this by composing `aleph'.toOrderEmbedding` with the order embedding that adds `ω`?

Reply:
What's the name of the latter?

Reply:
I don't know, but you can loogle for it using the lemma that characterizes its application.

If it doesn't exist, you could build it inline in this proof.

Reply:
Oh I found it, it's [`OrderEmbedding.addLeft`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Monoid/Basic.html#OrderEmbedding.addLeft).